proto/attest.proto

```diff
@@ -250,9 +257,35 @@ message PlatformPolicy {
   GCEConfidentialTechnology minimum_technology = 3;
 }
 
+// A policy about what parts of a RIM to compare against machine state as
+// reflected in a quote or (verified) event log. Reference measurements for
+// a component are expected to be addressable by the machine state's reported
+// digest, or otherwise presented as cached collateral with the attestation
+// itself. The method of delivery is vendor-specific.
+message RIMPolicy {
+  // If true, the signed measurement must be available and the target
```

Comment:
available where? IIUC, this is the attestation certificate table, but we state it.

Reply:
It can be in the certificate table, a UEFI variable, a local file, downloadable given a URI, or any other means that could be specified by a vendor. I've updated the wording a little.